src/boot/boot.janet

```diff
@@ -3374,6 +3377,11 @@
           (file/flush stdout)
           (file/read stdin :line buf))
         (def env (make-env))
+        (when-let [profile.janet (dyn :profilepath)] 
+            (def src (env :source))
```

Comment:
No need for this dance, I don't know why this is here. It doesn't do anything since :source should not be set by loading the profile.

Reply:
Also, why are you using `(dyn :profilepath)` instead of just using `jprofile` directly?

Reply:
Source seems to get set by merge-module, that problem I had with importing things went away when I did this fix.

Reply:
I'll change things for `jprofile` when I get back to my personal machine.

Reply:
So, revisiting this, I'm using `(dyn :profilepath)` here because the handler for the `-R` flag sets `(dyn :profilepath)` to null, but doesn't do the same to jprofile. I could change it to change jprofile as well, or get rid of jprofile as a def, but that's why I went down that road.

Reply:
I don't think :source is working how you think it is - `merge-module` does not touch `:source`